### PR TITLE
Support TS.INFO command

### PR DESCRIPTION
**Pull request by sazzad16**

### CODE REVIEW

**Comment by sazzad16** [comment]
@dengliming @yangbodong22011 Your comments in this PR would be much appreciated, especially regards to creation/usage of `DoublePrecision#parseFloatingPointNumber(String)`. Thanks :slightly_smiling_face:

File: src/main/java/redis/clients/jedis/timeseries/TSInfo.java [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+              List<List<Object>> rulesValueList = new ArrayList<>(rulesDataList.size());
+              rules = new HashMap<>(rulesDataList.size());
+              for (Object ruleData : rulesDataList) {
+                List<Object> encodedRule = (List<Object>) SafeEncoder.encodeObject(ruleData);
+                rulesValueList.add(encodedRule);
+                try {
+                  rules.put((String) encodedRule.get(0),
+                      new Rule((String) encodedRule.get(0), (Long) encodedRule.get(1),
+                          AggregationType.safeValueOf((String) encodedRule.get(2)), (Long) encodedRule.get(3)));
+                } catch (Exception e) { }
```
Comment (yangbodong22011) [inline src/main/java/redis/clients/jedis/timeseries/TSInfo.java:93]: I like to throw exceptions directly here

File: src/main/java/redis/clients/jedis/timeseries/TSInfo.java [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+              chunks = new ArrayList<>(chunksDataList.size());
+              for (Object chunkData : chunksDataList) {
+                Map<String, Object> chunk = BuilderFactory.ENCODED_OBJECT_MAP.build(chunkData);
+                chunksValueList.add(new HashMap<>(chunk));
+
+                if (chunk.containsKey(CHUNKS_BYTES_PER_SAMPLE_PROPERTY)) {
+                  try {
+                    chunk.put(CHUNKS_BYTES_PER_SAMPLE_PROPERTY,
+                        DoublePrecision.parseFloatingPointNumber((String) chunk.get(CHUNKS_BYTES_PER_SAMPLE_PROPERTY)));
+                  } catch (Exception e) { }
```
Comment (yangbodong22011) [inline src/main/java/redis/clients/jedis/timeseries/TSInfo.java:109]: ditto

File: src/main/java/redis/clients/jedis/timeseries/TSInfo.java (hunk 1 above)
Comment (sazzad16) [reply src/main/java/redis/clients/jedis/timeseries/TSInfo.java]: done

**Review by dengliming — APPROVED** [review]

**Pull request merged**